src/test/kotlin/lotto/domain/LotteryPaperFactoryTest.kt

```diff
@@ -0,0 +1,53 @@
+package lotto.domain
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class LotteryPaperFactoryTest {
+
+    private lateinit var lotteryPaperFactory: LotteryPaperFactory
+
+    @BeforeEach
+    fun setUp() {
+        lotteryPaperFactory = LotteryPaperFactory(RandomLottoNumberGenerationStrategy())
+    }
+
+    @Test
+    fun `1부터 45까지 숫자중 6개의 숫자를 선택해서 로또 번호를 생성한다`() {
+
+        val generatedLotteryPaper = lotteryPaperFactory.generateLotteryPaper(listOf())
+
+        Assertions.assertThat(generatedLotteryPaper.getLottoNumbers()).hasSize(6).allMatch { it.lottoNumber in 1..45 }
```

Comment:
given when then
 준비 실행 검증 단계를 나눠보면 어떨까요?
 테스트의 가독성도 올라가고, 무엇을 테스트하는지 명확해지기도 해요!
 ```suggestion

        // given
        val generatedLotteryPaper = lotteryPaperFactory.generateLotteryPaper(listOf())
        // when 
        val lottoNumber = generatedLotteryPaper.getLottoNumbers()
        val expectedSize = 6
        // then
        assertThat(lottoNumber)
             .hasSize(expectedSize)
             .allMatch { it.lottoNumber in 1..45 }
```